nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/gradient/LeakyReLUDerivative.java

```diff
@@ -31,23 +32,30 @@
 
 /**Leaky ReLU derivative. Default alpha = 0.01. Cutoff = 0
  */
+@NoArgsConstructor
 public class LeakyReLUDerivative extends BaseScalarOp {
     private double alpha = 0.01;
 
+    public LeakyReLUDerivative(SameDiff sameDiff, SDVariable i_v, SDVariable alpha) {
+
+    }
```

Comment:
Bad constructor (empty). + alpha should be double (change op definition in kotlin)

Reply:
Fixed.